actor/src/main/scala/com/evolutiongaming/akkaeffect/util/Terminated.scala

```diff
@@ -42,7 +42,10 @@ object Terminated {
             }
           }
 
-          actorRefOf(Props(actor())).use { _ => deferred.get }
+          actorRefOf(Props(actor())).use(_ => deferred.get).recover {
```

Comment:
Minor: could you please write chain methods from new line, that's much easier to read

```
actorRefOf(Props(actor()))
  .use(…)
  .recover { … }
```

Reply:
done